partner_event/models/event_registration.py

```diff
@@ -30,15 +34,14 @@ def create(self, vals):
                 [('email', '=ilike', email)], limit=1)
             event = event_model.browse(vals['event_id'])
             if partner:
-                partner_id = partner.id
                 vals['name'] = vals.get('name') or partner.name
                 vals['phone'] = vals.get('phone') or partner.phone
             elif event.create_partner:
                 # Create partner
```

Comment:
@chienandalu hello! This doesn't work if you register in a free event. The partner is not created. Should we fix it? I've seen two different behavior for the same workflow (registering a partner) depending if the register is executed in the backend or in the website